Index: src/drivers/infiniband/hw/qedr/qedr.h
===================================================================
--- src.orig/drivers/infiniband/hw/qedr/qedr.h	2017-12-05 18:31:22.892545884 +0100
+++ src/drivers/infiniband/hw/qedr/qedr.h	2017-12-05 18:31:56.901083706 +0100
@@ -41,7 +41,7 @@
 #include <linux/qed/roce_common.h>
 #include "qedr_hsi_rdma.h"
 
-#define QEDR_MODULE_VERSION	"8.10.10.0"
+#define QEDR_MODULE_VERSION	"8.10.10.0_dup7.4"
 #define QEDR_NODE_DESC "QLogic 579xx RoCE HCA"
 #define DP_NAME(dev) ((dev)->ibdev.name)
 
